src/newrelic/integration/parse.go

```diff
@@ -284,6 +285,13 @@ func parseConfig(test *Test, content []byte) error {
 }
 
 func parsePhpPackages(test *Test, content []byte) error {
+	// clean up provided string data if it is "null"
+	nullRE := regexp.MustCompile(`^\s*null\s*`)
+	if nullRE.Match(content) {
+		test.phpPackagesConfig = []byte("null")
+		return nil
+	}
```

Comment:
if the only need is to trim leading and trailing whitespaces, Go's `strings` package already has a means to do this without the need for regex.
Ex:
```
s := string(content[:])
```
In this use case `s` should now contain a string representation of `content` without leading or trailing whitespaces.

Reply:
See [this function](https://github.com/newrelic/newrelic-php-agent/pull/826/files#diff-0cca25c50dc0bef5ff2f46b4a27543cb92519898088734a6db5c2de837515846R276-R279) for an example of how you can apply this here

Reply:
using `TrimSpace` as opposed to regex follows the existing code convention from the functions above, but also makes it so we don't have to [import](https://github.com/newrelic/newrelic-php-agent/pull/826/files#diff-0cca25c50dc0bef5ff2f46b4a27543cb92519898088734a6db5c2de837515846R16) the `regexp` package for this one case. From a developer standpoint, I would also argue that `TrimSpace` is clearer as to the limited scope of what the function is doing at a glance than regex which can perform much more advanced operations that require analyzing the specific regex code.

Reply:
This [commit](https://github.com/newrelic/newrelic-php-agent/pull/826/commits/bf047013d599219a33fc42e64e122862de63e633) changes the parsing to use `TrimSpace` and is consistent with the other usage in this file.